src/metadata-builder/ClosureJunctionEntityMetadataBuilder.ts

```diff
@@ -57,7 +57,7 @@ export class ClosureJunctionEntityMetadataBuilder {
                 args: {
                     target: "",
                     mode: "virtual",
-                    propertyName: "descendant_" + primaryColumn.databaseName,
+                    propertyName: "descendant",
```

Comment:
I can merge this test however its not absolutely correct. Closure tables are not working with multiple primary keys and thats a problem they have right now